packages/react-components/src/components/RTE/RTERibbonButtons.tsx

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Theme } from '@fluentui/react';
+import { KnownRibbonButtonKey, LocalizedStrings, RibbonButton, getButtons } from 'roosterjs-react';
+import { ribbonButtonStyle, ribbonDividerStyle } from '../styles/RichTextEditor.styles';
+import { RTESendBoxStrings } from './RTESendBox';
+
+const dividerRibbonButton = (theme: Theme): RibbonButton<string> => {
+  return {
+    key: 'Divider',
+    iconName: 'RTEDividerIcon',
+    unlocalizedText: '',
+    onClick: () => {},
+    isDisabled: () => true,
+    commandBarProperties: {
+      buttonStyles: ribbonDividerStyle(theme)
+    }
+  };
+};
+
+const boldButton = (theme: Theme): RibbonButton<string> => {
+  return createKnownRibbonButton(KnownRibbonButtonKey.Bold, theme, 'RTEBoldButtonIcon');
+};
+
+const italicButton = (theme: Theme): RibbonButton<string> => {
+  return createKnownRibbonButton(KnownRibbonButtonKey.Italic, theme, 'RTEItalicButtonIcon');
+};
+
+const underlineButton = (theme: Theme): RibbonButton<string> => {
+  return createKnownRibbonButton(KnownRibbonButtonKey.Underline, theme, 'RTEUnderlineButtonIcon');
+};
+
+const bulletListButton = (theme: Theme): RibbonButton<string> => {
+  return createKnownRibbonButton(KnownRibbonButtonKey.BulletedList, theme, 'RTEBulletListButtonIcon');
+};
+
+const numberListButton = (theme: Theme): RibbonButton<string> => {
+  return createKnownRibbonButton(KnownRibbonButtonKey.NumberedList, theme, 'RTEtNumberListButtonIcon');
+};
+
+const indentIncreaseButton = (theme: Theme): RibbonButton<string> => {
+  return createKnownRibbonButton(KnownRibbonButtonKey.IncreaseIndent, theme, 'RTEIndentIncreaseButtonIcon');
+};
+
+const indentDecreaseButton = (theme: Theme): RibbonButton<string> => {
+  return createKnownRibbonButton(KnownRibbonButtonKey.DecreaseIndent, theme, 'RTEIndentDecreaseButtonIcon');
+};
+
+const createKnownRibbonButton = (key: KnownRibbonButtonKey, theme: Theme, icon: string): RibbonButton<string> => {
+  const button = getButtons([key])[0];
```

Comment:
Does getButtons function for sure returns an array with one item?

Reply:
I'm sending 1 item as a parameter and expect it to return the button item for that key

Reply:
> I'm sending 1 item as a parameter and expect it to return the button item for that key

Should we guard it a bit anyways? In case somehow RoosterJS didn't return us any item in the array?